docs/web-chat/api.mdx

```diff
@@ -137,6 +137,151 @@ You should call this API before a conversation starts. In other words, it works
+     last_name?: "string",
+     full_name?: "string",
+     display_name?: "string",
+     last_country?: "string",
+     last_city?: "string",
+     age?: "string",
+     lifetime_value?: "string",
+     user_persona?: "string",
+     user_vip_segment?: "string",
+     user_support_status?: "string"
+     last_active_date?: "string" // epoch time in ms/seconds
+     accepted_t_and_c?: "string" // true/false converted to string
+     preferred_language?:"string"
+     full_privacy_enabled?: "string"
+
+     // Non-primitives
+     new_tags?: Array<string>
+     tags_to_remove?: Array<string>
+     reset_tags?: Array<string>
+     custom_user_fields?: CustomUserFields
+}
+```
+
+Example:
+```js
+  // ... Some custom logic here that obtains user data
+  const full_name = "John Doe";
+
+  Helpshift("updateGlobalUserData", {full_name})
+```
+
+<Admonition type="info" title="Info">
+
+You can plug this API call into any place where you collect user data. However, be careful not to call it in any
+useEffect-like callbacks or frequently-firing events, since rate-limiting restrictions may kick in.
+
+Global user data is also referred to as **core attributes**.
+
+</Admonition>
+
+For a list of errors that can occur when using this API, refer [UPDATE_GLOBAL_USER_DATA_FAILED](#update-global-user-data-failed).
+
+## Update app user data {#update-app-user-data}
+
+This event is used in conjunction with the [new Identity system](/web-chat/users/#managing-users-with-the-identity-system) and is used to update the app data associated with a logged-in user.
+
+The following key/value pairs are accepted in this API call:
+```ts
+interface CustomUserFields {
+  [key: string]: string
+}
+
+interface AppUserData {
+     // Note that all primitive values must be coerced to string
+      app_version: "string",
+      user_vip_segment: "string",
+      sdk_version: "string",
+      language:"string",
+      app_rating: "string",
+      user_paying_segment: "string",
+      country: "string"
+      user_support_status: "string"
+      user_persona: "boolean string"
+      accepted_t_and_c: "string"
+      user_level: "string",
+      app_status: "string",
+      city: "string",
+      lifetime_value: "string",
+
+
+      // Non-primitives
+      custom_user_fields?: CustomUserFields
+}
+```
+
+Example:
+```js
+  // ... Some custom logic here that obtains user data
+  const app_rating = "5";
+
+  Helpshift("updateAppUserData", {app_rating})
+```
+
+<Admonition type="info" title="Info">
+
+You can plug this API call into any place where you collect user data. However, be careful not to call it in any
+useEffect-like callbacks or frequently-firing events, since rate-limiting restrictions may kick in.
+
+App user data is also referred to as **app attributes**.
+
+</Admonition>
+
+For a list of errors that can occur when using this API, refer [UPDATE_APP_USER_DATA_FAILED](#update-app-user-data-failed).
+
+## Add user identities {#add-user-identities}
```

Comment:
Let us make "Add user identities" the first thing even before the User Profile related information? Happy to discuss if I am missing context on the reason for adding it after the user profile information.